fastly.toml

```
@@ -1,4 +1,4 @@
-manifest_version = "0.1.0"
+manifest_version = "0.2.0"
```

```suggestion
manifest_version = "0.1.0"
```
My mistake! The manifest is still at v0.1.0